src/platform_impl/windows/event_loop.rs

```diff
@@ -1136,12 +1136,6 @@ unsafe fn public_window_callback_inner<T: 'static>(
                         }
                     }
                 }
-            } else if window_state
-                .window_flags
-                .contains(WindowFlags::ALWAYS_ON_BOTTOM)
-            {
-                let window_pos = &mut *(lparam as *mut WINDOWPOS);
-                window_pos.hwndInsertAfter = HWND_BOTTOM;
```

Comment:
why was this removed? I thought this is necessary to ensure it stays at the bottom

Reply:
According to this [SO answer](https://stackoverflow.com/a/65052538), it is but it is kind of an alternative way and the same is not done for always on top so I removed it for consistency.